README.md

```diff
@@ -43,30 +57,40 @@ showinf -autoscale `realpath /tmp/repo/Image/123/Binary/myimage`
 ```
 
 
-# Targeting multiple images
-
-Instead of using `Image` as a target, containers such as dataset or
-screens may be specified to target all their images. However, note that
-for plates the default server configuration disables file download.
+# Exporting images with metadata
 
-Additionally, specifying `-a` extends the targeted images to include all
-that are in the same fileset as any targeted image.
+The `-f` option supports `ome-tiff`, `ome-xml` and `tiff` to
+convert the data from original files into different formats.
 
+These files are generated from binary files and other
+metadata stored in OMERO, so this process may be slower than
+downloading.
 
-# Exporting images
+OME-TIFF includes pixel data, aquisition metadata and annotations.
+OME-XML does not include pixel data and TIFF is images only.
 
-A workaround to not being able to download plates is to instead export
-their images as TIFF. The `-f` option supports `tiff` and `ome-tiff`.
+The `tiff` option provides a workaround to not being able to download
+plates (see below).
```

Comment:
Maybe expand a bit more. I looked below and was confused not to see anything about using the `-f tiff` option.